### PR TITLE
chore(http): Move Ogg guide, HTTP method fixes

### PR DESCRIPTION
**Pull request by bsmth**
A couple of small fixes for HTTP section

__Changes:__
* Move Ogg guide into Media section
  * Some cleanup in Ogg guide 
* Browser compat information about HTTP methods.
* Update a few links
  * Node docs
  * web.dev

### CODE REVIEW

File: files/en-us/web/http/methods/patch/index.md [hunk 1]
```diff
@@ -117,6 +117,11 @@ ETag: "e0023aa4f"
 
 {{Specifications}}
 
+## Browser compatibility
+
+The browser doesn't use the `PATCH` method for user-initiated actions, so "browser compatibility" doesn't apply.
+Developers can set this request method using `fetch()`.
```
Comment (hamishwillee) [inline files/en-us/web/http/methods/patch/index.md:123]:
```suggestion
Developers can set this request method using [`fetch()`](/en-US/docs/Web/API/Window/fetch).
```

File: files/en-us/web/http/methods/delete/index.md [hunk 2]
```diff
@@ -111,7 +111,8 @@ Content-Length: 1234
 
 ## Browser compatibility
 
-{{Compat}}
+The browser doesn't use the `DELETE` method for user-initiated actions, so "browser compatibility" doesn't apply.
+Developers can set this request method using `fetch()`.
```
Comment (hamishwillee) [inline files/en-us/web/http/methods/delete/index.md:115]:
```suggestion
Developers can set this request method using [`fetch()`](/en-US/docs/Web/API/Window/fetch).
```

File: files/en-us/web/http/methods/put/index.md [hunk 3]
```diff
@@ -90,7 +90,8 @@ Content-Location: /existing.html
 
 ## Browser compatibility
 
-{{Compat}}
+The browser doesn't use the `PUT` method for user-initiated actions, so "browser compatibility" doesn't apply.
+Developers can set this request method using `fetch()`.
```
Comment (hamishwillee) [inline files/en-us/web/http/methods/put/index.md:94]:
```suggestion
Developers can set this request method using [`fetch()`](/en-US/docs/Web/API/Window/fetch).
```

File: files/en-us/web/http/methods/trace/index.md [hunk 4]
```diff
@@ -96,6 +96,11 @@ Accept: */*
 
 {{Specifications}}
 
+## Browser compatibility
+
+The browser doesn't use the `TRACE` method for user-initiated actions, so "browser compatibility" doesn't apply.
+Developers can set this request method using `fetch()`.
```
Comment (hamishwillee) [inline files/en-us/web/http/methods/trace/index.md:102]:
```suggestion
Developers can set this request method using [`fetch()`](/en-US/docs/Web/API/Window/fetch).
```

**Review by hamishwillee — APPROVED** [review]
Thanks @bsmth . Makes sense to me - merging.

Note https://github.com/mdn/content/pull/37026#discussion_r1865039611 - possible post-process activity

File: files/en-us/web/http/methods/delete/index.md [hunk 5]
```diff
@@ -111,7 +111,8 @@ Content-Length: 1234
 
 ## Browser compatibility
 
-{{Compat}}
+The browser doesn't use the `DELETE` method for user-initiated actions, so "browser compatibility" doesn't apply.
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/http/methods/delete/index.md:114]: > But I wouldn't have changed these without first working to get agreement and removal in BCD. Maybe you already did that? If not, can you kick it off?

Earlier in the year we discussed HTTP statuses and headers in BCD where support means "nothing breaks" -> the browser doesn't set them in requests and doesn't act on them in responses, but application-specific logic can be done using fetch(). I'll add this to the BCD call notes for this week to make sure this prose is okay, so thanks for flagging it 👍🏻

**Pull request merged**